preload/prefetch-load-event.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<title>Ensures that prefetch respects cache headers</title>
```

Comment:
Wrong title. You can omit the `<title>` if there's only a single test, to avoid these sorts of mismatches.